### PR TITLE
Add MSSQL connector with test cases

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
Add connector for Microsoft SQL Server

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/1752

### Example code
```
import ballerinax/mssql.driver as _;
import ballerinax/java.jdbc;

string url = "jdbc:sqlserver://localhost:1433;";
string user = "sa";
string password = "Test123#";

public function main() returns error? {
    jdbc:Client dbClient = check new (url, user, password);
    
    _ = check dbClient->execute("DROP DATABASE IF EXISTS TEST_DB");
    _ = check dbClient->execute("CREATE DATABASE TEST_DB");
    _ = check dbClient->execute("USE TEST_DB");

    _ = check dbClient->execute(`
        CREATE TABLE ExactNumericTypes (
            id INT NOT NULL IDENTITY PRIMARY KEY,
            smallint_type SMALLINT,
            int_type INT,
            tinyint_type TINYINT,
            bigint_type BIGINT,
            decimal_type DECIMAL,
            numeric_type NUMERIC,
        ); 
    `);
}
```

### CODE REVIEW

File: tests/batch-execute-query-test.bal [hunk 1]
```diff
@@ -0,0 +1,188 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (niveathika) [inline tests/batch-execute-query-test.bal:1]: No need to have all test cases here, only connection.bal test cases are enough. The point of test cases is to identify the driver is working thats all.

Comment (kaneeldias) [reply tests/batch-execute-query-test.bal]: Fixed in 2f32c83

File: tests/utils.bal [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    jdbc:Client dbClient;
+    if database is () {
+        dbClient = check new (url, user, password);
+    } else {
+        dbClient = check new (url + "databaseName=" + database, user, password);
+    }
+    return dbClient;
+}
+
+function queryMssqlClient(string|sql:ParameterizedQuery sqlQuery, typedesc<record {}>? resultType = (), string? database = ())
```
Comment (niveathika) [inline tests/utils.bal:34]: Are these functions used?

Comment (kaneeldias) [reply tests/utils.bal]: No they are not. All util functions removed in cf71558

**Review by niveathika — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      - uses: actions/checkout@v2
+      - name: Ballerina Build
+        uses: ballerina-platform/ballerina-action/@slbeta2
+        with:
+          args:
+            build -c --skip-tests
+        env:
+          DB_HOST: ${{ secrets.DB_HOST }}
+          DB_USER: ${{ secrets.DB_USER }}
+          DB_PASSWORD: ${{ secrets.DB_PASSWORD }}
```
Comment (daneshk) [inline .github/workflows/ci.yml:27]:
```suggestion
          DB_PASSWORD: ${{ secrets.DB_PASSWORD }}
          
```

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+          DB_HOST: ${{ secrets.DB_HOST }}
+          DB_USER: ${{ secrets.DB_USER }}
+          DB_PASSWORD: ${{ secrets.DB_PASSWORD }}
+      - name: Ballerina Push
+        uses: ballerina-platform/ballerina-action/@slbeta1
+        with:
+          args:
+            push
+        env:
+          BALLERINA_CENTRAL_ACCESS_TOKEN: ${{ secrets.BALLERINA_CENTRAL_ACCESS_TOKEN }}
```
Comment (daneshk) [inline .github/workflows/release.yml:29]:
```suggestion
          BALLERINA_CENTRAL_ACCESS_TOKEN: ${{ secrets.BALLERINA_CENTRAL_ACCESS_TOKEN }}
          
```

File: .github/workflows/ci.yml (hunk 3 above)
Comment (kaneeldias) [reply .github/workflows/ci.yml]: Fixed in ec9cb48

File: .github/workflows/release.yml (hunk 4 above)
Comment (kaneeldias) [reply .github/workflows/release.yml]: Fixed in ec9cb48

Comment (kaneeldias) [reply .github/workflows/release.yml]: Fixed in ec9cb48

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Ballerina Build
+        uses: ballerina-platform/ballerina-action/@slbeta2
```
Comment (daneshk) [inline .github/workflows/ci.yml:20]:
```suggestion
        uses: ballerina-platform/ballerina-action/@slbeta1
```

File: .github/workflows/release.yml [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Ballerina Build
+        uses: ballerina-platform/ballerina-action/@slbeta2
```
Comment (daneshk) [inline .github/workflows/release.yml:15]:
```suggestion
        uses: ballerina-platform/ballerina-action/@slbeta1
```

File: README.md [hunk 7]
```diff
@@ -1,2 +1,45 @@
...
+
+1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
+
+    * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+
+    * [OpenJDK](https://adoptopenjdk.net/)
+
+      > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+
+2. Download and install [Ballerina SL Beta 2](https://ballerina.io/).
```
Comment (daneshk) [inline README.md:20]:
```suggestion
2. Download and install [Ballerina SL Beta 1](https://ballerina.io/).
```

Comment (kaneeldias) [reply README.md]: Fixed in 5456a72

File: .github/workflows/release.yml (hunk 6 above)
Comment (kaneeldias) [reply .github/workflows/release.yml]: Fixed in 5456a72

File: .github/workflows/ci.yml (hunk 5 above)
Comment (kaneeldias) [reply .github/workflows/ci.yml]: Fixed in a078857

**Review by daneshk — APPROVED** [review]
LGTM

**Pull request merged**